corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -89,7 +89,11 @@
                 $loading = $main.find(".hq-loading").remove(),
                 $loadingError = $main.find(".hq-loading-error").remove();
 
-            $('#demand-releases').on('show.bs.tab', function () {
+            $('#demand-releases').on('show.bs.tab', _.throttle(function () {
```

Comment:
When I was testing this every click execute the listener twice. I verified that the listener is only added once and that the events for each execution were identical. Couldn't figure out why the 2nd execution was happening.